vision/use-cases/brochure-creation-using-imagen/brochure_creation_using_imagen.ipynb

```diff
@@ -340,11 +334,58 @@
         "]"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "id": "ccb4daf5",
+      "metadata": {
+        "id": "33418dee67f7"
+      },
+      "source": [
+        "### Multi-language Support Functions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 6,
+      "id": "779e7d77",
+      "metadata": {
+        "id": "6e2f96dfba28"
+      },
+      "outputs": [],
+      "source": [
+        "def get_supported_languages(project_id):\n",
+        "    client = translate.TranslationServiceClient()\n",
+        "    location = \"global\"\n",
+        "    parent = f\"projects/{PROJECT_ID}/locations/{location}\"\n",
+        "    response = client.get_supported_languages(\n",
+        "        request={\"parent\": parent, \"display_language_code\": \"en\"}\n",
+        "    )\n",
+        "    languages = response.languages\n",
+        "    language_dict = {\n",
+        "        language.language_code: language.display_name for language in languages\n",
+        "    }\n",
```

Comment:
I would recommend using the [`SupportedLanguage`](https://cloud.google.com/python/docs/reference/translate/latest/google.cloud.translate_v3.types.SupportedLanguages) class to maintain type safety.